Specs/FMDBSwift/0.0.2/FMDBSwift.podspec.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "FMDBSwift",
+  "version": "0.0.2",
+  "license": "Apache License, Version 2.0",
+  "summary": "Modified FMDB version to remove non modular headers (Swift Compatible)",
+  "homepage": "",
+  "authors": {
+    "Rafael Ring": "rafael.ring@taqtile.com"
```

Comment:
@felipesabino 
why is `Rafael Ring` the author if it uses a fork from another user?